src/components/stillage/StillageList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useEffect } from 'react'
+import { styled } from '@linaria/react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
+
+import StillageItem from './StillageItem'
+import { RootState } from '../../redux/store'
+import { clearStillageList, setStillageList } from '../../redux/slices/stillageSlice'
+import { api } from '../../api'
+
+const StyledList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  @media screen and (min-width: 820px) {
+    flex-direction: row;
+    flex-wrap: wrap;
+    li {
+      flex: 0 1 48.5%;
+    }
+  }
+  @media screen and (min-width: 1440px) {
+    gap: 20px;
+  }
+`
+
+function StillageList() {
+  const { id: stillageId } = useParams()
+  const dispatch = useDispatch()
+
+  const fetchData = useCallback(async () => {
+    const shelves = await api.shelves({ stillage: stillageId })
+    if (Array.isArray(shelves)) {
+      dispatch(setStillageList(shelves))
+    }
+  }, [])
```

Comment:
Don't you need to add `stillageId` in dependency array?

Reply:
Yes, I think I should've done it